app/src/main/java/org/schabi/newpipe/fragments/list/BaseListInfoFragment.java

```diff
@@ -257,10 +261,13 @@ protected void setInitialData(final int sid, final String u, final String title)
     private void dynamicallyShowErrorPanelOrSnackbar(final ErrorInfo errorInfo) {
         if (infoListAdapter.getItemCount() == 0) {
             // show error panel only if no items already visible
+
```

Comment:
Please remove those empty lines you added

Reply:
Thanks a lot.
Any more suggestions are most welcome.
PlaylistController need to be modified such that if no videos and somehow there is a way user is able to interact with the controller then app should not crash .